src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -193,7 +194,7 @@ public TopicIngestionProperties getIngestionProps(String topic) {
         return topicsToIngestionProps.get(topic);
     }
 
-    private void validateTableMappings(KustoSinkConfig config) {
+    public void validateTableMappings(KustoSinkConfig config) {
```

Comment:
```suggestion
    void validateTableMappings(KustoSinkConfig config) {
```
This way it's package protected and will be visible to respective test class.

Reply:
done